frontend/src/packages/projects/projects-api.ts

```diff
@@ -0,0 +1,34 @@
+import { type ProjectGetAllResponseDto } from 'shared/build/index.js';
+import { ProjectsApiPath } from 'shared/build/index.js';
```

Comment:
same

Reply:
why was it resolved without any commeny @dimapopovych is right?

Reply:
isn't resolved